src/promptflow/tests/test_configs/flows/llm_tool/echo.py

```diff
@@ -0,0 +1,13 @@
+from promptflow import tool
+
+# The inputs section will change based on the arguments of the tool function, after you save the code
```

Comment:
These comments are useless in test cases.

Reply:
Refined.